modules/API/API.js

```diff
@@ -155,6 +155,18 @@ function initCommands() {
         'avatar-url': avatarUrl => {
             sendAnalytics(createApiEvent('avatar.url.changed'));
             APP.conference.changeLocalAvatarUrl(avatarUrl);
+        },
+        'send-endpoint-text-message': (to, text) => {
+            sendAnalytics(createApiEvent('send.endpoint.message'));
```

Comment:
If large volumes of data are going to be sent with this mechanism, sending an analytics event for each message seems excessive

Reply:
Removed the analytics sending. Will ask today on standup if it s needed or not.